src/bson_mapper/CMakeLists.txt

```diff
@@ -119,6 +119,13 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
         REGEX ".+/test" EXCLUDE
 )
 
+install(DIRECTORY ${CMAKE_SOURCE_DIR}/src/third_party/cereal/include/cereal/
+    DESTINATION "include/cereal"
```

Comment:
The way it's currently written, it might override someone's system cereal headers which could be different from ours. (or worse, our headers may get overwritten by a modified version that changes the behavior of the bson_mapper).